build.cabal.sh

```diff
@@ -61,6 +61,11 @@ else
     # Initialize sandbox if necessary
     if ! ( "$CABAL" sandbox hc-pkg list > /dev/null 2>&1); then
         "$CABAL" sandbox init
+        ( cd ../libraries/Cabal/Cabal
```

Comment:
> Why not use cabal sandbox add-source?

To be honest, I never heard of `add-source` before. I will have a look at it later this day.